### PR TITLE
Remove org.elasticsearch exceptions

### PR DESCRIPTION
**Pull request by joschi**

### CODE REVIEW

File: graylog2-server/src/main/java/org/graylog2/indexer/ranges/MongoIndexRangeService.java [hunk 1]
```diff
@@ -211,7 +210,7 @@ public void handleIndexReopening(IndicesReopenedEvent event) {
             indices.waitForRecovery(index);
 
             final Retryer<IndexRange> retryer = RetryerBuilder.<IndexRange>newBuilder()
-                .retryIfException(input -> !(input instanceof IndexClosedException))
+                // .retryIfException(input -> !(input instanceof IndexClosedException))
```
Comment (dennisoelkers) [inline graylog2-server/src/main/java/org/graylog2/indexer/ranges/MongoIndexRangeService.java]: Can we just remove the line if it's not used anymore?

File: graylog2-server/src/main/java/org/graylog2/indexer/ranges/MongoIndexRangeService.java [hunk 2]
```diff
@@ -221,11 +220,13 @@ public void handleIndexReopening(IndicesReopenedEvent event) {
                 indexRange = retryer.call(() -> calculateRange(index));
                 auditEventSender.success(AuditActor.system(nodeId), ES_INDEX_RANGE_CREATE, ImmutableMap.of("index_name", index));
             } catch (Exception e) {
+                /*
                 if (e.getCause() instanceof IndexClosedException) {
                     LOG.debug("Couldn't calculate index range for closed index \"" + index + "\"", e.getCause());
                     auditEventSender.failure(AuditActor.system(nodeId), ES_INDEX_RANGE_CREATE, ImmutableMap.of("index_name", index));
                     return;
                 }
+                */
```
Comment (dennisoelkers) [inline graylog2-server/src/main/java/org/graylog2/indexer/ranges/MongoIndexRangeService.java]: Same here.

File: graylog2-server/src/main/java/org/graylog2/indexer/ranges/MongoIndexRangeService.java (hunk 1 above)
Comment (joschi) [inline graylog2-server/src/main/java/org/graylog2/indexer/ranges/MongoIndexRangeService.java]: ✅ Unintentionally forgot to remove this.

File: graylog2-server/src/main/java/org/graylog2/indexer/ranges/MongoIndexRangeService.java (hunk 2 above)
Comment (joschi) [inline graylog2-server/src/main/java/org/graylog2/indexer/ranges/MongoIndexRangeService.java]: ✅

**Pull request merged**